classes/fields/file.php

```diff
@@ -631,7 +632,11 @@ public function save( $value, $id = null, $name = null, $options = null, $fields
 
 			// Update the attachment if it the data array is not still empty.
 			if ( ! empty( $attachment_data ) ) {
-				$attachment_data['ID'] = $id;
+				$attachment_data['ID']        = $id;
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 8 spaces

<sub>hash: 08204a30b93e01a34923fbcdad47bcdd</sub>